youtube_dl/YoutubeDL.py

```diff
@@ -1190,9 +1190,9 @@ def _calc_headers(self, info_dict):
         if add_headers:
             res.update(add_headers)
 
-        cookies = self._calc_cookies(info_dict)
-        if cookies:
-            res['Cookie'] = cookies
+        #cookies = self._calc_cookies(info_dict)
+        #if cookies:
+        #    res['Cookie'] = cookies
```

Comment:
Why comment out these lines? They are used for at least cookie-based authentication.